avaframe/in3Utils/fileHandlerUtils.py

```diff
@@ -213,19 +255,21 @@ def makeSimDict(inputDir, varPar=''):
         varPar = 'Mu'
 
     # Make dictionary of input data info
-    data = {'files': [], 'names': [], 'resType': [], 'simType': [],
+    data = {'files': [], 'names': [], 'resType': [], 'simType': [], 'simName' : [],
             'modelType' : [], 'releaseArea': [], 'cellSize': [], varPar: []}
 
     for m in range(len(datafiles)):
         data['files'].append(datafiles[m])
         name = os.path.splitext(os.path.basename(datafiles[m]))[0]
+        data['simName']
```

Comment:
What is this needed for?

Reply:
simName is e.g. used in the generate report function and the plot all peak files function

Reply:
But this line does not do anything!? Or is it meant for initialising the variable?

Reply:
Correct, setting simName appears some lines later, so I removed it here